assets/scss/forums/discussion.scss

```diff
@@ -1,19 +1,19 @@
 .bb-single-reply-list {
```

Comment:
Recommend a //TODO at the top of this file and number the lines that need updating in a future release.

Reply:
good thinking. Done.